openshift/release/generate-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source $(dirname $0)/resolve.sh
+
+root_dir=$(dirname $0)/../..
+
+release=$(yq '.project.tag' openshift/project.yaml)
+release=${release/knative-/}
+
+echo "Release: $release"
+
+./openshift/generate.sh
+
+artifacts_dir="openshift/release/artifacts"
+rm -rf $artifacts_dir
+mkdir -p $artifacts_dir
+
+image_prefix="registry.ci.openshift.org/openshift/knative-${release}:knative-"
```

Comment:
```suggestion
image_prefix="registry.ci.openshift.org/openshift/knative-${release}:knative-backstage-plugins-"
```

Reply:
It was like that exactly, but then I changed it.

Image name that way is like: 

`registry.ci.openshift.org/openshift/knative-v1.12:knative-backstage-plugins-backstage-plugin-backend-eventmesh`

vs

`registry.ci.openshift.org/openshift/knative-v1.12:knative-backstage-plugin-backend-eventmesh`

@pierDipi

Reply:
I think that's because of your regex (that I suggested to remove)

Reply:
The expected final name should be `registry.ci.openshift.org/openshift/knative-v1.12:knative-backstage-plugins-eventmesh` because:
- `knative-backstage-plugins-` is the image prefix in project.yaml
- `eventmesh` is the directory name in `backends/cmd/...`